cert/init.sls

```diff
@@ -22,7 +22,9 @@ cert_packages:
 
   {% set cert = data.get('cert', False) %}
   {% set key = data.get('key', False) %}
-  {% set cert_user = data.get('cert_user', map.cert_user) %}
+  {% set cert_ext = data.get('cert_ext', map.cert_ext) %}
+  {% set key_ext = data.get('key_ext', map.key_ext) %}
+  {% set cert_user = data.get('cert_user', map.cert_user) %}  
```

Comment:
@grandmasterv Looks like you added couple of spaces to the end of this line.

Reply:
No problem.  Fixed in latest commit.  Is this a best practice thing to avoid spaces at the ends of lines?  Sorry, I'm new to contributing.

Reply:
@grandmasterv No problem at all, welcome to contributing.  Yes, trailing spaces are avoided at all times.